### PR TITLE
Adding new Object Display topic

### PR DESCRIPTION
**Pull request by randall-Mysten**
## Description 

Describe the changes or additions included in this PR.

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: doc/src/build/sui-object-display.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+## Typical objects with data duplication 
+
+A common case with in-game items is a large number of similar objects grouped by some criteria. It is important to optimize their size and the cost to mint and update them. Typically, a game uses a single source image or URL per group or item criteria. Storing the source image inside of every object is not optimal. 
+In some cases, in-game items are minted by users when a game allows them or when they purchase an in-game item. To enable this, some IPFS/Arweave metadata must be created and stored in advance. This requires additional logic that is usually not related to the in-game properties of the item.
+
+The following example demonstrates how to create a Capy:
+
+```rust
+module capy::capy_items {
+   /// A wearable Capy Item. For some items there can be an
```
Comment (ronny-mysten) [inline doc/src/build/sui-object-display.md:184]:
```suggestion
   /// A wearable Capy item. For some items there can be an
```

File: doc/src/build/sui-object-display.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+   /// unlimited supply. And items with the same name are identical
```
Comment (ronny-mysten) [inline doc/src/build/sui-object-display.md:185]:
```suggestion
   /// unlimited supply. And items with the same name are identical.
```

File: doc/src/build/sui-object-display.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+}
+```
+
+## Unique objects with dynamic representation
+
+Sui Capys use dynamic image generation. When a Capy is born, its attributes determine the Capy’s appearance, such as color or pattern. When a user puts an item on a Capy, the Capy’s appearance changes. When users put multiple items on a Capy, there’s a chance of a bonus for a combination of items.
+
+To implement this, the Capys game API service refreshes the image in response to a user-initiated change. The URL for a Capy is a template with the `capy.id`. But storing the full URL - as well as other fields in the Capy object due to their diverse population - also leads to users paying for excess storage and increased gas fees. 
+
+The following example demonstrates to implement dynamic image generation:
```
Comment (ronny-mysten) [inline doc/src/build/sui-object-display.md:199]:
```suggestion
The following example demonstrates how to implement dynamic image generation:
```

File: doc/src/build/sui-object-display.md [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+   /// new genes and manage the Capy Market
+   struct CapyManagerCap has key, store {
+id: UID }
+}
+```
+
+## Typical objects with data duplication 
+
+A common case with in-game items is a large number of similar objects grouped by some criteria. It is important to optimize their size and the cost to mint and update them. Typically, a game uses a single source image or URL per group or item criteria. Storing the source image inside of every object is not optimal. 
+In some cases, in-game items are minted by users when a game allows them or when they purchase an in-game item. To enable this, some IPFS/Arweave metadata must be created and stored in advance. This requires additional logic that is usually not related to the in-game properties of the item.
```
Comment (ronny-mysten) [inline doc/src/build/sui-object-display.md:178]:
```suggestion
In some cases, users mint in-game items when a game allows them or when they purchase an in-game item. To enable this, some IPFS/Arweave metadata must be created and stored in advance. This requires additional logic that is usually not related to the in-game properties of the item.
```
Passive voice but not sure who is doing the creating and storing of metadata.

**Review by ronny-mysten — APPROVED** [review]

File: doc/src/build/sui-object-display.md [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+```rust
+module sui::display {
+    /// Update the version of Display and emit an event
+    public fun update_version(self: &mut Display) { /* ... */ }
+}
+```
+
+## Sui utility objects
+
+In Sui, utility objects enable authorization for Capabilities. Almost all modules have features that can be accessed only with the required Capability. Generic modules allow one Capability per Application, such as a marketplace. Some Capabilities mark ownership of a shared object on-chain, or access the shared data from another account.
```
[the remark this answers is not in the document]
Comment (randall-Mysten) [reply doc/src/build/sui-object-display.md]: updated, ty, I didn't change the content from source doc

File: doc/src/build/sui-object-display.md [hunk 6]
```diff
@@ -0,0 +1,228 @@
+---
+title: Sui Object Display Standard
+---
+
+The Sui Object Display standard is a template engine that allows for off-chain management of display configurations for a type. With it, you can substitute data for an object into a template string. The standard doesn’t limit the fields you can set. You can use the  `{property}` syntax to access all object properties, and then insert them as a part of the template string.
```
Comment (damirka) [inline doc/src/build/sui-object-display.md:5]: Its should be a reverse - on-chain management for off-chain representation (display).

File: doc/src/build/sui-object-display.md [hunk 7]
```diff
@@ -0,0 +1,228 @@
...
+
+You can use the `sui::display` module to define the display properties for a `Publisher` object you own. For more information about `Publisher` objects, see [Publisher](https://examples.sui.io/basics/publisher.html) topic in *Sui Move by Example*.
```
Comment (damirka) [inline doc/src/build/sui-object-display.md:7]: The opposite - `Publisher` object is required to set `sui::display` for a type.

File: doc/src/build/sui-object-display.md [hunk 8]
```diff
@@ -0,0 +1,228 @@
...
+
+In Sui Move, `Display<T>` represents an object that specifies a set of named templates for the type `T`. For example, for a type `0x2::capy::Capy` the display syntax is:  `Display<0x2::capy::Capy>`. 
+
+Sui Full nodes process all objects of the type `T` by matching the `Display` definition, and return the processed result when you query an object.
```
Comment (damirka) [inline doc/src/build/sui-object-display.md:11]: When query an object with `{ showDisplay: true }` setting in the query.

File: doc/src/build/sui-object-display.md [hunk 9]
```diff
@@ -0,0 +1,228 @@
...
+
+After you create the `Display`, you can modify it. The following code sample demonstrates how to modify a `Display`:
+
+```rust
+module sui::display {
+    /// Sets multiple fields at once
+    public fun add_multiple(
+        self: &mut Display,
+        keys: vector<String>,
+        values: vector<String
```
Comment (damirka) [inline doc/src/build/sui-object-display.md:137]: Something that we got wrong in the examples and needs fixing there as well.
```suggestion
        values: vector<String>
```

File: doc/src/build/sui-object-display.md [hunk 10]
```diff
@@ -0,0 +1,228 @@
...
+In some cases, users mint in-game items when a game allows them or when they purchase an in-game item. To enable this, some IPFS/Arweave metadata must be created and stored in advance. This requires additional logic that is usually not related to the in-game properties of the item.
+
+The following example demonstrates how to create a Capy:
+
+```rust
+module capy::capy_items {
+   /// A wearable Capy item. For some items there can be an
+   /// unlimited supply. And items with the same name are identical.
+   struct CapyItem has key, store {
+id: UID,
```
Comment (damirka) [inline doc/src/build/sui-object-display.md:187]:
```suggestion
        id: UID,
```

File: doc/src/build/sui-object-display.md [hunk 11]
```diff
@@ -0,0 +1,228 @@
...
+With capabilities, it is important to provide a meaningful description of objects to facilitate user interface implementation. This helps avoid accidentally transferring the wrong object when objects are similar. It also provides a user-friendly description of items that users see.
+
+The following example demonstrates how to create a capy capability:
+
+```rust
+module capy::utility {
+   /// A capability which grants Capy Manager permission to add
+   /// new genes and manage the Capy Market
+   struct CapyManagerCap has key, store {
+id: UID }
```
Comment (damirka) [inline doc/src/build/sui-object-display.md:171]:
```suggestion
        id: UID
    }
```

File: doc/src/build/sui-object-display.md [hunk 12]
```diff
@@ -0,0 +1,228 @@
...
+
+The following example demonstrates how to implement dynamic image generation:
+
+```rust
+module capy::capy {
+   /// A Capy - very diverse object with different combination
+   /// of genes. Created dynamically + for images a dynamic SVG
+   /// generation is used.
+   struct Capy has key, store {
+id: UID,
```
Comment (damirka) [inline doc/src/build/sui-object-display.md:207]:
```suggestion
        id: UID,
```

**Review by damirka — APPROVED** [review]
Left a few comments but approving to unblock.

**Pull request merged**